src/components/Footer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '../style/footer.css';
+import facebookIcon from '../images/facebook.svg';
+import twitterIcon from '../images/twitter.svg';
+import youtubeIcon from '../images/youtube.svg';
+import instagramIcon from '../images/instagram.svg';
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <p className="footer-text">@codeit - 2023</p>
+      <div className="footer-link">
+        <a href="/privacy">
+          <p className="footer-privacy">Privacy Policy</p>
+        </a>
+        <a href="/FAQ">
+          <p className="footer-faq">FAQ</p>
+        </a>
+      </div>
+      <div className="footer-icon">
+        <a href="https://facebook.com/" target="_blank">
+          <img src={facebookIcon} />
+        </a>
+        <a href="https://twitter.com/" target="_blank">
+          <img src={twitterIcon} />
+        </a>
+        <a href="https://youtube.com/" target="_blank">
+          <img src={youtubeIcon} />
+        </a>
+        <a href="https://instagram.com/" target="_blank">
+          <img src={instagramIcon} />
+        </a>
```

Comment:
```suggestion
       {SNS_LIST.map(({ name, href, src }) => {
              return (
                <a key={name} href={href} target="_blank" rel="noreferrer">
                  <img src={src} alt={`${name} icon`} width="20" height="20" />
                </a>
              );
            })}
```

```js

const SNS_LIST = [
  { name: 'facebook', href: 'https://www.facebook.com', src: facebookIcon },
  { name: 'twitter', href: 'https://twitter.com', src: twitterIcon },
  { name: 'youtube', href: 'https://youtube.com', src: youtubeIcon },
  { name: 'instagram', href: 'https://www.instagram.com', src: instagramIcon },
];
```

이런식으로 배열을 구성한 뒤 map 돌리는 로직으로 구현하게 되면 추후 SNS 아이콘이 추가/수정/삭제될때 대응하기 편해집니다.

Reply:
객체로 정리하면 나중에 정말 추가 및 삭제하는 등 유지보수에 용이하군요 .. 앞으로는 배열로 구성하는 연습을 하겠습니다 !